docs/manual/regex-checker.tex

```diff
@@ -63,7 +63,7 @@
 \end{description}
 
 The subtyping hierarchy of the Regex Checker's qualifiers is shown in
-Figures~\ref{fig-regex-hierarchy-1} and \ref{fig-regex-hierarchy-2}.
+Figures~\ref{fig-regex-hierarchy-1} and~\ref{fig-regex-hierarchy-2}.
```

Comment:
If you change existing labels, that will break existing links into the manual.
It's OK to add new labels (such as for new figures or sections), but try to avoid changing existing ones.

Reply:
There was only one place where the original `@Regex` hierarchy image was linked and I did replace it. I built the manual and the links were working too. Should I revert anyways or is it fine?

Reply:
My concern is not just with links within the manual.  I was thinking about external links into the manual, such as from webpages, issues, and email threads.